aten/src/ATen/cuda/CublasHandlePool.cpp

```diff
@@ -48,6 +48,41 @@ void destroyCublasLtHandle(cublasLtHandle_t handle) {
 }
 
 using CuBlasLtPoolType = DeviceThreadHandlePool<cublasLtHandle_t, createCublasLtHandle, destroyCublasLtHandle>;
+
```

Comment:
Just want to make sure this is not an oversight.

The `#if defined(USE_ROCM)` is on Line 32 and include CUDA code. I understand that hipify will convert these to hip/roc calls. So Lines 33-50 are not needed for NV?